README.md

```diff
@@ -2,9 +2,11 @@
 
 Helper packages for JS projects
 
+<!-- Managed by @eternagame/nx-plugin - readme prolog -->
```

Comment:
Nit: "prologue"

Reply:
Good catch - spellcheck didn't complain because Prolog is a thing. :D I'll fix it elsewhere in the code as well